examples/mnist-pytorch/README.rst

```diff
@@ -23,108 +28,96 @@ Clone this repository, locate into this directory:
    git clone https://github.com/scaleoutsystems/fedn.git
    cd fedn/examples/mnist-pytorch
 
-Start a pseudo-distributed FEDn network using docker-compose:
+Create the package using script:
 
 .. code-block::
 
-   docker-compose -f ../../docker-compose.yaml up
-
-This starts up the needed backend services MongoDB and Minio, the API Server and one Combiner. 
-You can verify the deployment using these urls: 
-
-- API Server: http://localhost:8092/get_controller_status
-- Minio: http://localhost:9000
-- Mongo Express: http://localhost:8081
+   bin/build.sh
 
-Next, we will prepare the client. A key concept in FEDn is the compute package - 
-a code bundle that contains entrypoints for training and (optionally) validating a model update on the client. 
+This should create a file 'package.tgz' in the project folder.
 
-Locate into 'examples/mnist-pytorch' and familiarize yourself with the project structure. The entrypoints
-are defined in 'client/entrypoint'. The dependencies needed in the client environment are specified in 
-'requirements.txt'. For convenience, we have provided utility scripts to set up a virtual environment.    
+Upload the package to FEDn Studio project. 
 
-Start by initializing a virtual enviroment with all of the required dependencies for this project.
+Start the client using the client.yaml file from FEDn Studio.
 
 .. code-block::
 
-   bin/init_venv.sh
-
-Next create the compute package and a seed model:
+   pip install fedn 
+   export FEDN_AUTH_SCHEME=Bearer
+   export FEDN_PACKAGE_EXTRACT_DIR=package
+   fedn run client -in client.yaml --name client1 --secure=True --force-ssl
 
-.. code-block::
 
-   bin/build.sh
+Upload the initial model to FEDn Studio project. The seed.npz file is created when you run start the client and is found in ./data/models/seed.npz
```

Comment:
I understand that the intention is to make it "one-shot" to start the example, but I worry that it would feel a bit illogical that the seed is created by the client upon startup? It feels that that step belongs to. a "model initiator" part of the flow?